R-package/src/lightgbm_R.cpp

```diff
@@ -1060,8 +1224,15 @@ static const R_CallMethodDef CallEntries[] = {
   {"LGBM_BoosterPredictForFile_R"     , (DL_FUNC) &LGBM_BoosterPredictForFile_R     , 10},
   {"LGBM_BoosterCalcNumPredict_R"     , (DL_FUNC) &LGBM_BoosterCalcNumPredict_R     , 8},
   {"LGBM_BoosterPredictForCSC_R"      , (DL_FUNC) &LGBM_BoosterPredictForCSC_R      , 14},
-  {"LGBM_BoosterPredictForMat_R"      , (DL_FUNC) &LGBM_BoosterPredictForMat_R      , 11},
+  {"LGBM_BoosterPredictForCSR_R"      , (DL_FUNC) &LGBM_BoosterPredictForCSR_R      , 12},
```

Comment:
Please preserve the existing formatting of this structure, where all elements are vertically aligned. Please also 

If these new, longer names like `LGBM_BoosterPredictForCSRSingleRowFastInit_R` means that other lines not strictly related to this PR also need to have their indentation changes, that is fine.